Update README.md (#372)
dianna-ai/dianna: README.md

It's built by, with and for (academic) researchers and research software engineers working on machine learning projects.

## Why DIANNA? 

> DIANNA software is addressing needs of both (X)AI researchers and mostly the various domains scientists who are using or will use AI models for their research without being experts in (X)AI. DIANNA is future-proof: one of the very few XAI library supporting the [Open Neural Network Exchange (ONNX)](https://onnx.ai/) format. 

Reviewer: ```suggestion
DIANNA software is facilitates both (X)AI researchers and domain scientists who want to use AI models for their research without being experts in (X)AI. DIANNA is future-proof: one of the very few XAI library supporting the [Open Neural Network Exchange (ONNX)](https://onnx.ai/) format. 
```
Some changes to make it read easier (I hope).

After studying the vast XAI landscape we have made choices in the parts of the [XAI Taxonomy](https://doi.org/10.3390/make3030032) on which methods, data modalities and problems types to focus. Our choices, based on the largest usage in scientific literature, are shown graphically in the XAI taxonomy below:

![XAI_taxonomy](https://user-images.githubusercontent.com/3244249/196441854-24c8c6b7-2364-4cb3-90b9-1c63e3bdc345.png)

The key points of DIANNA:

* Provides an easy-to-use interface for non (X)AI experts
* Implements well-known XAI methods (LIME, RISE and Kernal SHAP) chosen by systematic and objective evaluation criteria
* Supports the de-facto standard format for neural network models - ONNX.
* Includes clear instructions for export/conversions from Tensorflow, Pytorch, Keras and scikit-learn to ONNX.

> * Supports both images and text data modalities. Time series is work in progress, tabular data and even embeddings support is planned.

Reviewer: ```suggestion
* Supports images and text data modalities. Work on both time series and embeddings modalities is in progress. Support for tabular modality is planned for later.
```
